tr/cs-221-states-models.md

```diff
@@ -0,0 +1,980 @@
+
+<br>
+
+
+**86. Model-free Monte Carlo ― The model-free Monte Carlo method aims at directly estimating Qπ, as follows:**
+
+&#10230; Model içermeyen Monte Carlo ― Model içermeyen Monte Carlo yöntemi aşağıdaki şekilde doğrudan Qπ'yi tahmin etmeyi amaçlar:
+
+<br>
+
+
+**87. Qπ(s,a)=average of ut where st−1=s,at=a**
+
+&#10230; Qπ(s,a)= ortalama ut , st−1=s ve at=a olduğunda
+
+<br>
+
+
+**88. where ut denotes the utility starting at step t of a given episode.**
+
+&#10230; ut belirli bir bölümün t anında başlayan faydayı ifade etmektedir.
+
+<br>
+
+
+**89. Remark: model-free Monte Carlo is said to be on-policy, because the estimated value is dependent on the policy π used to generate the data.**
+
+&#10230; Not: model içermeyen Monte Carlo'nun politikaya dahil olduğu söyleniyor, çünkü tahmini değer veriyi üretmek için kullanılan π politikasına bağlıdır.
+
+<br>
+
+
+**90. Equivalent formulation - By introducing the constant η=11+(#updates to (s,a)) and for each (s,a,u) of the training set, the update rule of model-free Monte Carlo has a convex combination formulation:**
+
+&#10230; Eşdeğer formülasyon - Sabit tanımı η=11+(#güncelleme sayısı (s,a) ) ve eğitim kümesinin her bir (s,a,u) üçlemesi için, model içermeyen Monte Carlo'nun güncelleme kuralı dışbükey bir kombinasyon formülasyonuna sahiptir:
+
+<br>
+
+
+**91. as well as a stochastic gradient formulation:**
+
+&#10230; olasılıksal bayır formülasyonu yanında:
+
+<br>
+
+
+**92. SARSA ― State-action-reward-state-action (SARSA) is a boostrapping method estimating Qπ by using both raw data and estimates as part of the update rule. For each (s,a,r,s′,a′), we have:**
+
+&#10230; SARSA ― Durum-eylem-ödül-durum-eylem (State-Action-Reward-State-Action - SARSA), hem ham verileri hem de güncelleme kuralının bir parçası olarak tahminleri kullanarak Qπ'yi tahmin eden bir destekleme yöntemidir. Her bir (s,a,r,s′,a′) için:
+
+<br>
+
+
+**93. Remark: the SARSA estimate is updated on the fly as opposed to the model-free Monte Carlo one where the estimate can only be updated at the end of the episode.**
+
+&#10230; Not: the SARSA tahmini, tahminin yalnızca bölüm sonunda güncellenebildiği model içermeyen Monte Carlo yönteminin aksine anında güncellenir.
+
+<br>
+
+
+**94. Q-learning ― Q-learning is an off-policy algorithm that produces an estimate for Qopt. On each (s,a,r,s′,a′), we have:**
+
+&#10230; Q-öğrenme ― Q-öğrenme, Qopt için tahmin üreten politikaya dahil olmayan bir algoritmadır. Her bir (s,a,r,s′,a′) için:
+
+<br>
+
+
+**95. Epsilon-greedy ― The epsilon-greedy policy is an algorithm that balances exploration with probability ϵ and exploitation with probability 1−ϵ. For a given state s, the policy πact is computed as follows:**
+
+&#10230; Epsilon-açgözlü ― Epsilon-açgözlü politika, ϵ olasılıkla araştırmayı ve 1−ϵ olasılıkla sömürüyü dengeleyen bir algoritmadır. Her bir s durumu için, politika, πact, aşağıdaki şekilde hesaplanır:
```

Comment:
Her bir s durumu için, πact politikası aşağıdaki şekilde hesaplanır:

Reply:
OK